test_util/test_runner.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python3
+"""Module for running integration_test.py inside of a remote cluster
```

Comment:
How hard would it be to leverage this in postflight during the installation process?